firefox-ios/nimbus-features/deeplinkOptimizationRefactorFeature.yaml

```diff
@@ -0,0 +1,18 @@
+# The configuration for the deeplinkOptimizationRefactorFeature feature
+features:
+  deeplink-optimization-refactor-feature:
+    description: >
+      The feature flag to manage the rollout of deeplink optimization feature.
+    variables:
+      enabled:
+        description: >
+          Variable description
+        type: Boolean
+        default: false
+    defaults:
+      - channel: beta
+        value:
+          enabled: false
```

Comment:
Do we want to do a rollout for this change? Maybe we should have it ON in beta too? Testing with Testflight is a necessary step for sure since deeplinks can behave differently with application that have been put in background for a while